### ISSUE TITLE
Unhandled Promise Rejection on first run

### ISSUE THREAD

**Issue by innocenzi** [tinkerun/tinkerun#12]
I tried `Tinkerun.0.2.1.exe` from the release page.

![](https://i.imgur.com/UBvmpF0.png)

```
Unhandled Promise Rejection
TypeError: Cannot read property 'setMenu' of undefined
    at App.<anonymous> (C:/Users/ENZOIN~1/AppData/Local/Temp/1stNgUsKDUMMZJJ9hU5AWocDTMC/resources/app.asar/main/index.js:53589:12)
```

---

I'm running Windows 10 Enterprise:

```
Edition	Windows 10 Enterprise
Version	20H2
OS build	19042.985
Experience	Windows Feature Experience Pack 120.2212.2020.0
```

**Comment by renatobcj** [comment]
Same here

**Comment by billyct** [comment]
I don’t have a PC, 🤣 so I often don’t test on windows.

it fixed in [v0.2.2](https://github.com/tinkerun/tinkerun/releases/tag/v0.2.2)
